doc/classes/EditorTranslationParserPlugin.xml

```diff
@@ -5,30 +5,37 @@
 	</brief_description>
 	<description>
 		Plugins are registered via [method EditorPlugin.add_translation_parser_plugin] method. To define the parsing and string extraction logic, override the [method parse_file] method in script.
+		Add the extracted strings to argument [code]msgids[/code] or [code]msgids_context_plural[/code] if context or plural is used.
 		The extracted strings will be written into a POT file selected by user under "POT Generation" in "Localization" tab in "Project Settings" menu.
-		Below shows an example of a custom parser that extracts strings in a CSV file to write into a POT.
+		Below shows an example of a custom parser that extracts strings from a CSV file to write into a POT.
 		[codeblock]
 		tool
 		extends EditorTranslationParserPlugin
 
 
-		func parse_file(path, extracted_strings):
+		func parse_file(path, msgids, msgids_context_plural):
 		    var file = File.new()
 		    file.open(path, File.READ)
 		    var text = file.get_as_text()
 		    var split_strs = text.split(",", false, 0)
 		    for s in split_strs:
-		        extracted_strings.append(s)
+		        msgids.append(s)
 		        #print("Extracted string: " + s)
 
 
 		func get_recognized_extensions():
 		    return ["csv"]
 		[/codeblock]
+		To add a translatable string associated with context or plural, add it to [code]msgids_context_plural[/code]:
+		[codeblock]
+		msgids_ctx_plural.append(["Test 1", "context", "test 1 plurals"]) # This will add a message with msgid "Test 1", msgctxt "context", and msgid_plural "test 1 plurals".
+		msgids_ctx_plural.append(["A test without context", "", "plurals"]) # This will add a message with msgid "A test without context" and msgid_plural "plurals".
+		msgids_ctx_plural.append(["Only with context", "a friendly context", ""]) # This will add a message with msgid "Only with context" and msgctxt "a friendly context".
```

Comment:
```suggestion
		msgids_context_plural.append(["Test 1", "context", "test 1 plurals"]) # This will add a message with msgid "Test 1", msgctxt "context", and msgid_plural "test 1 plurals".
		msgids_context_plural.append(["A test without context", "", "plurals"]) # This will add a message with msgid "A test without context" and msgid_plural "plurals".
		msgids_context_plural.append(["Only with context", "a friendly context", ""]) # This will add a message with msgid "Only with context" and msgctxt "a friendly context".
```
The line are also a bit long, I'd suggest putting the comments before each line of code.